src/lightning_lite/strategies/fsdp.py

```diff
@@ -91,7 +90,7 @@ def __init__(
         precision: Optional[Precision] = None,
         process_group_backend: Optional[str] = None,
         timeout: Optional[timedelta] = default_pg_timeout,
-        cpu_offload: Optional["CPUOffload"] = None,
+        cpu_offload: Optional[Union[bool, "CPUOffload"]] = None,
```

Comment:
```suggestion
        cpu_offload: Union[bool, "CPUOffload", None] = None,
```

Reply:
Sure, I'll make the change, but it's not the same style we have in the code base